### DISCUSSION TITLE
Out with the old check-in, and in with the NEW ☀️  [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#114621] [New to GitHub]
![Header of New to GitHub category check-in shows a purple starry background with shooting stars and clouds next to an octocat in a beret, striped shirt, and red pants holding a stack of books.](https://github.com/community/community/assets/92824518/3ff6f15b-be36-4fac-a2c8-e1bc3ba92918)

We’re almost out of March, but before we officially close out the month, we’ve got one more New to GitHub check-in to celebrate our members, share learning opportunities, and stay up to date on what is new in GitHub and the Community!  

### 🌟Community Superstars: A round of applause for these community citizens 👏 
* @Waresnew: Shared proper formatting for saving a SSH key in [their comment](https://github.com/orgs/community/discussions/113537#discussioncomment-8871782). Great formatting and great explanation!  
* @rdmccart: Is experiencing no conflicts with Git, but conflicts in GitHub, in [their post](https://github.com/orgs/community/discussions/112811) they clearly stated the steps they have taken, shared screenshots, and embodied a spirit of curiosity. They’re still looking for an assistance and if you have Git/GitHub experience you can lend, [check out their post](https://github.com/orgs/community/discussions/112811). 
* @0x013FA11: [came through with the command needed to change an author of a commit](https://github.com/orgs/community/discussions/113721) - short and sweet, but with so many commands to remember, it is nice knowing a fellow community member has your back. 

### ⬆️ Level up: Resources to take your skills to new heights
* **How to build your first Devcontainer**: Join @scubaninja and @marcduiker as they show you how to create a full-featured development environment using a devcontainer in [their hands-on video tutorial](https://www.youtube.com/watch?v=C_5tDWsWSj0). You will learn how to create one from scratch and understand how easy it is to then get started working on any code base. Devcontainers will enable yourself and your development teams to code from any device and to get started writing code in minutes!  
* **4 things you didn’t know you could do with GitHub Projects**: Unlock the secret to organization and collaboration magic with our [GitHub Projects tips and tricks roundup](https://github.blog/2024-03-21-insider-newsletter-digest-4-things-you-didnt-know-you-could-do-with-github-projects/) from @saraverdi.
* **Hard and soft skills for developers coding in the age of AI**: While AI revolutionizes software development, it still relies on developers to pilot its use. [In this blogpost](https://github.blog/2024-03-07-hard-and-soft-skills-for-developers-coding-in-the-age-of-ai/), we’ll cover the skills that developers need to have for navigating this new AI-powered coding frontier.

### 🆕 What’s new? The latest news and releases from GitHub and the community. 
* **Nominate an Outstanding Community Member** ⭐: Have you been helped by a community superstar that you’d like to see celebrated? [Nominate them in our Discussion](https://github.com/orgs/community/discussions/114271) by April 1st @ 10am EST.
* **GitHub Certifications are now available worldwide**: GitHub Certifications are now accessible globally! Elevate your tech profile by earning a certification that demonstrates your GitHub prowess. @LadyKerr and @a-a-ron dive into the program's ins and outs and how you can achieve GitHub Certification in [their video](https://www.youtube.com/watch?v=M3IZLIvr7GA). Expand your GitHub expertise and open the door to new possibilities!

As we close out March, share something new you learned or an accomplishment from the past month in GitHub!

**Comment by mst-rajatmishra** [comment]
"Thanks for the update! I've learned about scheduling reminders on GitHub recently.
